roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -53,13 +66,13 @@
 
     "network_device": "field:port:device_owner=~^network:",
     "create_port": "",
-    "create_port:device_owner": "not rule:network_device or rule:admin_or_network_owner or rule:context_is_advsvc",
-    "create_port:mac_address": "rule:admin_or_network_owner or rule:context_is_advsvc",
-    "create_port:fixed_ips": "rule:admin_or_network_owner or rule:context_is_advsvc",
-    "create_port:port_security_enabled": "rule:admin_or_network_owner or rule:context_is_advsvc",
+    "create_port:device_owner": "not rule:network_device or rule:context_is_advsvc or rule:admin_or_network_owner",
+    "create_port:mac_address": "rule:context_is_advsvc or rule:admin_or_network_owner",
```

Comment:
The reordering here doesn't affect the rules/behaviors. It is written like this in the community files and has been updated here so we don't get false hits when comparing.